bool start
geometry_msgs/Point next
---
bool get_request